stdlib/Profile/test/runtests.jl

```diff
@@ -279,7 +279,9 @@ end
 end
 
 @testset "HeapSnapshot" begin
-    fname = read(`$(Base.julia_cmd()) --startup-file=no -e "using Profile; print(Profile.take_heap_snapshot())"`, String)
+    fname = cd(mktempdir()) do
```

Comment:
this should have cleanup somewhere

Reply:
The file is `rm`-ed manually at the end of the testset and the tempdir is by default marked for cleanup.  Isn't that enough?